src/App.js

```diff
@@ -1,13 +1,14 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
Why is `useEffect` being imported here?

Reply:
Used it for getting data into the app. Will delete.